docs/developer/architecture/service-observability.md

```
@@ -114,3 +114,115 @@ func (t *TracedService) ListCertificateSigningRequests(ctx context.Context, p ap
 - Standardizes status and error reporting
 - Keeps all span logic consistent and easy to audit
 
+---
+
+## Graceful Shutdown Integration
+
+All FlightCtl services implement standardized graceful shutdown that works seamlessly with OpenTelemetry tracing. The shutdown process ensures spans are properly flushed before service termination.
+
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)